packages/stacked_cli/lib/src/constants/command_constants.dart

```diff
@@ -3,6 +3,8 @@
 const String ksDart = 'dart';
 const String ksFlutter = 'flutter';
 const String ksCreate = 'create';
+const String ksOrganization = "--org";
```

Comment:
This is the only constant used in the command line that has -- in front of it. Please follow the convention of the rest of the option implementations. Look at template for reference.